src/core/PAGE_main.jl

```diff
@@ -331,15 +331,14 @@ function perturb_marginal_page_emissions!(base::Model, marginal::Model, gas::Sym
         marginal_emissions_growth[i, :] = pulse
 
         # Marginal emissions model
-        md = marginal.mi.md 
+        md = marginal.mi.md
         update_param!(md, :marginal_emissions_growth, marginal_emissions_growth)    # this updates the marginal_emissions_growth parameter that both :er_CO2emissionsgrowth and :AbatementCostsCO2_er_emissionsgrowth are connected to from the PAGE_marginal_emissions comp
```

Comment:
Just a note that this call might be a little faster, but less safe/less checks, if you called `update_param!(mi, ...)` but if things work fine now don't rock the boat :)